src/unity/extensions/additional_sframe_utilities.cpp

```diff
@@ -16,6 +16,55 @@
 using namespace turi;
 
 
+template <typename T>
+void copy_image_to_memory(const image_type& img, T *outptr,
+                          const std::vector<size_t>& outstrides,
```

Comment:
The name `outstrides` is a bit confusing - it at first suggests to me that it's an out parameter (like `outptr`). But looking at the code, it looks like an in parameter, that is meant to be used with the out parameter `outptr`. What about calling it something like `strides_for_outptr` instead?

Reply:
Yes, it's the strides associated with `outptr`. These names are taken from `sframe_load_to_numpy` in the same file, so let's change them together if we do. In that case, any opinion on `field_length` too? Would simply `strides` be clear?